src/main/kotlin/de/cyface/uploader/DefaultUploader.kt

```diff
@@ -171,81 +171,98 @@ class DefaultUploader(private val apiEndpoint: String) : Uploader {
      * We wrap errors with [UploadFailed] so that the caller can handle this without crashing.
      * This way the SDK's `SyncPerformer` can determine if the sync should be repeated.
      */
-    @Suppress("ComplexMethod")
     private fun handleUploadException(exception: Exception): Nothing {
```

Comment:
Had to reduce the complexity of this method, so I extracted the network specific exception into an own method